.github/workflows/test.yml

```diff
@@ -63,7 +63,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        version: ['1.1.0', '1.2.4', '1.3.4', '2.2.0', '2.4.0', '2.6.0']
+        version: ['1.1.0', '1.2.4', '1.3.4', '2.2.0', '2.4.0', '2.6.0', '2.6.0']
```

Comment:
You mean 2.8.0? ;)

Reply:
Surely, thanks @dblock !

Reply:
I don't just blinding merge all your PRs @reta :)

Reply:
:smile: thanks @dblock !